cirq/ops/common_channels.py

```diff
@@ -187,3 +189,528 @@ def depolarize(p: float) -> DepolarizingChannel:
         ValueError if p is not a valid probability.
     """
     return DepolarizingChannel(p)
+
+
+class GeneralizedAmplitudeDampingChannel(raw_types.Gate):
+    """ The generalized amplitude damping channel."""
+
+    def __init__(self, p, gamma) -> None:
+
+        """
+        This channel models the effect of dissipation to the environment
+        at possibly non zero tempurature. Where gamma is the damping rate
```

Comment:
You don't need to define gamma and p here, since they're listed in the args.

Reply:
think I've fixed everywhere ?